### DISCUSSION TITLE
Combined domain plots

### DISCUSSION THREAD

**Discussion by andreabono** [GSI-CS-CO/chart-fx#577] [Q&A] [answered]
Hello everybody!!  
Is there any way to get a chart with a **combined domain XY plot**?  
What I need is a chart with **N synchronized time series** (so, the time on the x axis is the same) plotted one below another like in a VBox.  See the attached picture.  
Is there any documentation about it?  
  
![sample](https://user-images.githubusercontent.com/50195930/228841486-eaf7c2e4-ee1f-4232-a49b-5657deb27bac.jpg)

**Comment by wirew0rm** [comment, marked as answer]
I don't think there's a simple out of the box way to do that at the moment, but I think the issue has been discussed before. probably the best bet at the moment is to have synchronized hidden axes and fixing the widths of the y-axes to force all charts to the same with. Another approach would be a custom axis which consists of an vbox of multiple axes. which can then be used with different renderers.

We're about to restructure a lot of the layouting code and while this is probably not inside of the scope there, my hope is that afterwards it will be easier to implement something like this.

**Comment by lacgit** [comment]
Is it possible for a chart to refer the X-Axis of another chart?  Then get display position and get value by position can be synchronized.  And then possibly can zoom in/out together.  Then its only a matter for individual charts to visualize their X-Axes.  Is my thought doable?
